### PR TITLE
add mariner trusted launch skeleton

### PR DESCRIPTION
**Pull request by anujmaheshwari1**
**What type of PR is this?**

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Requirements**:
- [ ] uses [conventional commit messages](https://www.conventionalcommits.org/)
  
- [ ] includes documentation
- [ ] adds unit tests
- [ ] tested upgrade from previous version

**Special notes for your reviewer**:

**Release note**:
```
none
```

### CODE REVIEW

File: vhdbuilder/packer/convert-sig-to-classic-storage-account-blob.sh [hunk 1]
```diff
@@ -34,7 +34,7 @@ fi
 sig_resource_id="/subscriptions/${SUBSCRIPTION_ID}/resourceGroups/${RESOURCE_GROUP_NAME}/providers/Microsoft.Compute/galleries/${SIG_GALLERY_NAME}/images/${SIG_IMAGE_NAME}/versions/${SIG_IMAGE_VERSION}"
 disk_resource_id="/subscriptions/${SUBSCRIPTION_ID}/resourceGroups/${RESOURCE_GROUP_NAME}/providers/Microsoft.Compute/disks/${CAPTURED_SIG_VERSION}"
 
-if [[ ${OS_TYPE} == "Linux" && ${ENABLE_TRUSTED_LAUNCH} == "True" ]]; then
+if [[ ("$OS_TYPE" == "Linux" || "$OS_SKU" == "CBLMariner") && "$ENABLE_TRUSTED_LAUNCH" == "True" ]]; then
```
Comment (alexeldeib) [inline vhdbuilder/packer/convert-sig-to-classic-storage-account-blob.sh:37]: the extra condition on CBLMariner adds nothing?

Comment (anujmaheshwari1) [reply vhdbuilder/packer/convert-sig-to-classic-storage-account-blob.sh:37]: duh uh, yes. for some reason, I thought it was Ubuntu and not Linux. thanks!

File: vhdbuilder/packer/init-variables.sh [hunk 2]
```diff
@@ -109,13 +109,17 @@ if [[ "${MODE}" == "linuxVhdMode" ]]; then
 		if [[ "${OS_SKU}" == "Ubuntu" ]]; then
```
Comment (cameronmeissner) [inline vhdbuilder/packer/init-variables.sh:109]: nit: can now probably un-nest this to:

` if [[ "${OS_SKU}" == "Ubuntu" && "${IMG_SKU}" == "20_04-lts-cvm" ]]`

Comment (anujmaheshwari1) [reply vhdbuilder/packer/init-variables.sh]: fair enough, addressed!

**Review by cameronmeissner — APPROVED** [review]

**Pull request merged**